var/spack/repos/builtin/packages/postgresql/package.py

```diff
@@ -125,3 +125,22 @@ def setup_dependent_run_environment(self, env, dependent_spec):
             env.prepend_path('TCLLIBPATH', self.prefix.lib)
         if '+python' in spec:
             env.prepend_path('PYTHONPATH', self.prefix.lib)
+
+    @property
+    def libs(self):
+        stat_libs = ['libecpg_compat', 'libecpg', 'libpgcommon',
+                     'libpgcommon_shlib', 'libpgfeutils', 'libpgport',
+                     'libpgport_shlib', 'libpgtypes', 'libpq']
+        fl_stat = find_libraries(stat_libs, self.prefix, shared=False,
+                                 recursive=True)
+
+        dyn_libs = ['libecpg_compat', 'libecpg', 'libpgtypes', 'libpq',
+                    'libpqwalreceiver', 'plpgsql', 'pgoutput']
+        if '+perl' in self.spec:   dyn_libs.append('plperl')  # noqa: E701
+        if '+python' in self.spec: dyn_libs.append('plpython')  # noqa: E701
+        if '+tcl' in self.spec:    dyn_libs.append('pltcl')  # noqa: E701
```

Comment:
Can you remove the `#noqa`?

Reply:
without it, I can't do a single-liner if statement since flake complains.

Reply:
Sure, two lines are fine.

Reply:
It would be nice to allow for single liner 'if's in my opinion. in any case I just pushed the fix. Thanks.

Reply:
We try to stick to PEP-8 whenever possible. There's always more than one way to do things, but style guides like PEP-8 prevent people from arguing about formatting details that don't actually matter. Soon we'll be using `black` to autoformat the whole library so all you'll have to do is ask spackbot to run `black` and you don't need to worry about anything else.